packages/pnpm/src/cmd/store.ts

```diff
@@ -64,3 +76,67 @@ async function statusCmd (opts: PnpmOptions) {
 
   throw new StoreStatusError(modifiedPkgs)
 }
+
+/**
+ * Uses archy to output package usages in a directory-tree like format.
+ * @param packageUsagesResponses a list of package usages, one per query
+ */
+function prettyPrintUsages (packageUsagesResponses: FindPackageUsagesResponse[]): void {
```

Comment:
I think it would be fine to call the type `PackageUsage` instead of `FindPackageUsagesResponse`